simplesensor/config/logging.conf

```diff
@@ -34,13 +34,6 @@ level=DEBUG
 formatter=simpleClientDebugFormatter
 args=(sys.stdout,)
 
-[handler_file]
```

Comment:
why are you taking file out?  why not use it and just leave the handler defined in case someone later might want to flip the switch back on in line 5 [handlers] section